pkg/hooks/loader.go

```diff
@@ -237,6 +251,17 @@ func (h *Hook) GetDestinationInfo(srcPort uint16) (*structs.DestInfo, error) {
 	return &destInfo, nil
 }
 
+// this function is used in case of running keploy tests along with unit tests of application
```

Comment:
please use the write syntax/format to document the functions in golang. ref: https://go.dev/src/go/doc/example.go